src/newViews/Tagging/Export/NewExport/ExportSelection/index.tsx

```diff
@@ -0,0 +1,462 @@
+}
+
+export interface SelectedLead extends Lead {
+    selected: boolean;
+}
+interface Props {
+    className?: string;
+    projectRole: {
+        exportPermissions?: {
+            'create_only_unprotected'?: boolean;
+        };
+    };
+    projectId: number;
+}
+
+function EntriesExportSelection(props: Props) {
+    const {
+        className,
+        projectId,
+        projectRole,
+    } = props;
+
+    const filterOnlyUnprotected = !!projectRole?.exportPermissions?.create_only_unprotected;
+    const [queryTitle, setQueryTitle] = useState<string>();
+    const [previewId, setPreviewId] = useState<number | undefined>(undefined);
+    const [activeExportTypeKey, setActiveExportTypeKey] = useState<ExportType>('word');
+    const [decoupledEntries, setDecoupledEntries] = useState<boolean>(true);
+    const [textWidgets, setTextWidgets] = useState<TreeSelectableWidget<string | number>[]>([]);
+    const [showGroups, setShowGroups] = useState<boolean>(true);
+    const [showEntryId, setShowEntryId] = useState<boolean>(true);
+    const [showAryDetails, setShowAryDetails] = useState<boolean>(true);
+    const [showAdditionalMetadata, setShowAdditionalMetadata] = useState<boolean>(true);
+    const [reportStructure, setReportStructure] = useState<ReportStructure[]>([]);
+    const [includeSubSector, setIncludeSubSector] = useState<boolean>(false);
+    const [isPreview, setIsPreview] = useState<boolean>(false);
+    const [selectedLeads, setSelectedLeads] = useState<number[]>([]);
+    const [selectAll, setSelectAll] = useState<boolean>(true);
+    const [filterValues, setFilterValues] = useState<SourceEntryFilter>({});
+
+    const [
+        reportStructureVariant,
+        setReportStructureVariant,
+    ] = useState<string>(SECTOR_FIRST);
+    const [
+        contextualWidgets,
+        setContextualWidgets,
+    ] = useState<TreeSelectableWidget<string | number>[]>([]);
+
+    const {
+        pending: analysisFrameworkPending,
+        response: analysisFramework,
+    } = useRequest<FrameworkFields>({
+        url: `server://projects/${projectId}/analysis-framework/`,
+        method: 'GET',
+        schemaName: 'analysisFramework',
+        onSuccess: (response) => {
+            const textWidgetList = getTextWidgetsFromFramework(response);
+            const contextualWidgetList = getContextualWidgetsFromFramework(response);
+            setTextWidgets(textWidgetList);
+            setContextualWidgets(contextualWidgetList);
+        },
+        failureHeader: _ts('export', 'afLabel'),
+    });
+
+    const geoOptionsRequestQueryParams = useMemo(() => ({
+        project: projectId,
+    }), [projectId]);
+
+    const {
+        pending: geoOptionsPending,
+        response: geoOptions,
+    } = useRequest<GeoOptions>({
+        url: 'server://geo-options/',
+        method: 'GET',
+        query: geoOptionsRequestQueryParams,
+        schemaName: 'geoOptions',
+        failureHeader: _ts('export', 'geoLabel'),
+    });
```

Comment:
Geo-options are not required in the new filter for now.

Reply:
@AdityaKhatri removed for now